lib/spider/pyproject.toml

```diff
@@ -61,9 +61,11 @@ nltk = "^3.8.1"
 pandas = "^2.1.3"
 plotly = "^5.18.0"
 python = "~3.11"
-python-dotenv = "^1.0.0"
 torch = "^2.1.1"
-transformers = "^4.36.2"
+transformers = "^4.35.2"
```

Comment:
Is there a reason not to take the latest with `^v4.36.2`?

Reply:
nope, I need to upgrade it 😵‍💫